samples/spring-framework-petclinic-12.2.1.4/README.md

```diff
@@ -14,148 +14,151 @@
 
 # Spring PetClinic Sample Application
 
-This repo is a fork of the [spring-projects/spring-framework-petclinic](https://github.com/spring-projects/spring-framework-petclinic).
+This repo is a fork of the [spring-projects/spring-framework-petclinic](https://github.com/spring-petclinic/spring-framework-petclinic/tree/5.3.x).
 
 This version is for the **Oracle WebLogic community** to deploy a Spring based application on a specific **WebLogic Domain** release.
 The example follows the same **3-layer architecture** (i.e. presentation --> service --> repository) included in spring-framework-petclinic version.
- 
 
-## Understanding the Spring Petclinic application with a few diagrams
+<!--
+## Understanding the Spring PetClinic application with a few diagrams
 
 [ TODO: Update with Diagrams ]
+ -->
 
-## Running petclinic
+## Running PetClinic
 
-### Build WebApplication Archive
+### Build the Web Application Archive (WAR) file
 ```bash
 git clone https://github.com/oracle-samples/weblogic-examples.git
 cd weblogic-examples/12_2_1_4/spring-petclinic
 ./mvnw clean package
 ```
 
-### Deploy to WebLogic Domain
+### Deploy to a WebLogic domain
+
+Use any of the following deployment tools for deploying the PetClinic application to a WebLogic domain.
 
 #### WebLogic Remote Console
 
-You can use WebLogic Remote Console to manage the deployment process of applications to WebLogic Server.
+You can use [WebLogic Remote Console](https://docs.oracle.com/en/middleware/fusion-middleware/weblogic-remote-console/administer/set-console.html) to manage the deployment process of applications to WebLogic Server. For more information, see [Deploying Applications](https://docs-uat.us.oracle.com/en/middleware/fusion-middleware/weblogic-remote-console/administer/deploying-applications.html) in the _Oracle WebLogic Remote Console Online Help_.
 
-For general information on the application deployment process, see [Understanding WebLogic Server Deployment](https://docs.oracle.com/pls/topic/lookup?ctx=en/middleware/fusion-middleware/weblogic-remote-console/administer&id=DEPGD-GUID-F6E8BF0B-FBCF-44D2-A33F-13C4EF2E0031) in Deploying Applications to Oracle WebLogic Server.
+For general information about the application deployment process, see [Understanding WebLogic Server Deployment](https://docs.oracle.com/pls/topic/lookup?ctx=en/middleware/fusion-middleware/weblogic-remote-console/administer&id=DEPGD-GUID-F6E8BF0B-FBCF-44D2-A33F-13C4EF2E0031) in _Deploying Applications to Oracle WebLogic Server_.
 
 Installing an application makes its physical file or directory known to WebLogic Server.
 
-This procedure applies to all of the deployment units listed in Supported Deployment Units. 
+This procedure applies to all of the deployment units listed in Supported Deployment Units.
 
-    In the Edit Tree, go to Deployments, then App Deployments.
+1. In the Edit Tree, go to Deployments, then App Deployments.
 
-    Select New.
+2. Select New.
```

Comment:
you can write `1.` for all bullet points that the markdown will automatically number for you